plugins/rikki/loungeviews/components/spoilerstoggle/default.htm

```diff
@@ -0,0 +1,32 @@
+<div>
+    <span>Show Spoilers</span>
```

Comment:
I think this should be about the same size vertically as the switch

Reply:
done